### DISCUSSION TITLE
Adblock test

### DISCUSSION THREAD

**Discussion by Spacewalker2** [StevenBlack/hosts#2493] [Ideas]
Maybe the logs button and this site can be of some help.

https://d3ward.github.io/toolz/adblock.html

**Comment by iam-py-test** [comment]
I'm sorry, but this website is quite inaccurate.
Some of the "adservers" it tests for aren't really adservers or trackers (old issue, but several of the domains are still tested for): https://github.com/d3ward/toolz/issues/31
Thanks

**Comment by Clemens-Ratte-Polle** [comment]
is there a list of tracking-ad-blocker test sites and therefore following a Top 10 of Adblockers ?
I use this: 
https://adblock.turtlecute.org/
